Resources/config/factories.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="cmf_resource.factory.filesystem" class="Symfony\Cmf\Bundle\ResourceBundle\Factory\FilesystemFactory">
+            <tag name="cmf_resource.repository_factory" alias="filesystem" />
+        </service>
+
+        <service id="cmf_resource.factory.doctrine_phpcr" class="Symfony\Cmf\Bundle\ResourceBundle\Factory\PhpcrFactory">
```

Comment:
This PR means that we don't have to. The reason we separate them elsewhere is so that the service definitions which reference "suggested" dependencies are only loaded when explicitly enabled.

On 13 May 2016 05:32:00 BST, Maximilian Berghoff <notifications@github.com> wrote:
>> @@ -0,0 +1,28 @@
>> +<?xml version="1.0" encoding="UTF-8"?>
>> +<container xmlns="http://symfony.com/schema/dic/services"
>> +    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
>> +    xsi:schemaLocation="http://symfony.com/schema/dic/services
>http://symfony.com/schema/dic/services/services-1.0.xsd">
>> +
>> +    <services>
>> +
>> +        <service id="cmf_resource.factory.filesystem"
>class="Symfony\Cmf\Bundle\ResourceBundle\Factory\FilesystemFactory">
>> +            <tag name="cmf_resource.repository_factory"
>alias="filesystem" />
>> +        </service>
>> +
>> +        <service id="cmf_resource.factory.doctrine_phpcr"
>class="Symfony\Cmf\Bundle\ResourceBundle\Factory\PhpcrFactory">
>
>We do not have to devide services definitions by its persistence layer
>in this bundle yet, right?
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/symfony-cmf/resource-bundle/pull/18/files/48e552f9c7e6f6d3844e50994774b400b0a69960#r63132386

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.